website/app/components/doc/page/banner.js

```diff
@@ -0,0 +1,29 @@
+/**
+ * Copyright (c) HashiCorp, Inc.
+ * SPDX-License-Identifier: MPL-2.0
+ */
+
+import Component from '@glimmer/component';
+import { action } from '@ember/object';
+import { tracked } from '@glimmer/tracking';
+
+const COOKIE_NAME = 'hide-banner';
```

Comment:
do we have GDPR laws that apply here if we start using cookies?

Reply:
We are not storing any kind of personal data, so I don't think so.